.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       # use `--frozen-lockfile` to fail immediately if the committed yarn.lock needs updates
       # https://yarnpkg.com/lang/en/docs/cli/install/#toc-yarn-install-frozen-lockfile
-      - run: yarn --frozen-lockfile
```

Comment:
This is not necessary. `yarn` recognizes ci environments and freezes the lockfile by itself.

Reply:
> 'Important: This documentation covers modern versions of Yarn. For 1.x docs, see classic.yarnpkg.com.'

We are using Yarn 1 here

Reply:
I did not know that `yarn@1` behaves so poorly. Thanks for the update. I removed this part of the PR. However I recommend using `yarn@3` soonish, it is much better than `yarn@1`.